lib/DDG/Spice/DebVersion.pm

```diff
@@ -0,0 +1,24 @@
+# Get the version of a package in various debian repositories
+package DDG::Spice::DebVersion;
+
+use Text::Trim;
+use DDG::Spice;
+
+triggers startend => "debian version", "debian versions", "deb ver", "debian ver";
+
+# spice from => '([^/]+)/?(?:([^/]+)/?(?:([^/]+)|)|)';
+spice to => 'http://sources.debian.net/api/src/$1/';
```

Comment:
It's not redirecting to HTTPS for me. :confused: But HTTPS works and it's always the better choice. Will change it.